### ISSUE TITLE
Increment should create both record and bin if the record does not yet exist

### ISSUE THREAD

**Issue by rbotzer** [aerospike/aerospike-client-php#30]
Similar to append(), increment() should create a new record with the bin set to an initial value if the record does not yet exist. This should follow the C client's as_operations_add_incr() behavior.

```php
<?php
$config = [ 'hosts' => [['addr'=>'192.168.119.3','port'=>3000]]];
$db = new Aerospike($config);
$k = $db->initKey('test', 'eg-set', 'eg-key');
$status = $db->increment($k, 2, 2);
var_dump($status);
$db->close();
```

**Comment by rbotzer** [comment]
If this requires dropping the 'initial_value' argument of increment() to perform in a single operation, that's fine.

**Comment by rbotzer** [comment]
Implemented in release >= 3.3.11
